docs/source/release/v6.4.0/mantidworkbench.rst

```diff
@@ -13,5 +13,11 @@ Bugfixes
 --------
 - In DrILL interface, scrolling down in the settings dialog no longer affects the comboxes
 - Cleaned up the appearance of the main window and the Indirect, Reflectometry, and Engineering GUIs on macOS.
+- Fixed where matplotlib 3.5 caused 3D plots to throw a warning in multiple places.
```

Comment:
Can we move these notes out to a separate file as per the annoucements in slack?

Reply:
My understanding is it should be fine as long as it is merged before Friday?